application/common/System.cpp

```diff
@@ -0,0 +1,152 @@
+#include "loom/script/loomscript.h"
+#include "loom/common/utils/utString.h"
+#include "loom/common/platform//platformThread.h"
+#include <iostream>
+#include <stdio.h>
+#include <stdlib.h>
+
+class System
```

Comment:
There is already a class called [Process](https://github.com/LoomSDK/LoomSDK/blob/dfe41a7b508b23c4695a40023ebd523b1c4ea655/sdk/src/system/Process.ls), perhaps this functionality should be added to that class?